nebula3/gclient/net/Connection.py

```diff
@@ -229,7 +229,10 @@ def execute_json_with_parameter(self, session_id, stmt, params):
         """
```

Comment:
Let's keep back compatibility to fix the doc string of the return type instead and keep the bytes as output for now.